packages/datadog-plugin-tedious/test/index.spec.js

```diff
@@ -0,0 +1,143 @@
+'use strict'
+
+const agent = require('../../dd-trace/test/plugins/agent')
+const plugin = require('../src')
+
+const MSSQL_USERNAME = 'sa'
+const MSSQL_PASSWORD = 'DD_HUNTER2'
+
+wrapIt()
+
+describe('Plugin', () => {
+  let tds
+  let tracer
+  let connection
+
+  withVersions(plugin, 'tedious', version => {
+    beforeEach(() => {
+      tracer = require('../../dd-trace')
+    })
+
+    describe('without configuration', () => {
+      let config
+
+      beforeEach(() => {
+        config = {
+          server: 'localhost',
+          options: {
+            database: 'master'
+          }
+        }
+        if (version === '3.0.0') {
```

Comment:
Use `semver` with the exact version when this change was done to avoid the condition potentially breaking in the future. Alternatively and if this is supported, assign everything to a single config object.